applications/luci-app-isc-dhcp/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2022 Jaymin Patel <jem.patel@gmail.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for the ISC DHCP
+LUCI_DEPENDS:=+luci-base +isc-dhcp-server-ipv4 +isc-dhcp-dyndns-ipv4
+LUCI_PKGARCH:=all
+PKG_LICENSE:=GPL-2.0
```

Comment:
I overlooked this in my previous review. The comment at the top states Apache 2.0 (as it is common for all LuCI packages), here you specify GPL-2.0. Any particular reason for the GPL exception?

Reply:
Updated.